engine/runtime/src/test/java/org/enso/example/TestClass.java

```diff
@@ -19,6 +20,32 @@ public static long add(long a, long b) {
     return a + b;
   }
 
+  public static double doubleArrayAverage(double[] arr) {
+    var sum = 0.0;
+    for (int i = 0; i < arr.length; i++) {
+      sum += arr[i];
+    }
+    return sum / arr.length;
+  }
+
+  public static double numberArrayAverage(Number[] arr) {
```

Comment:
Should one need to distinguish between various Enso `Number` types, one can accept `Number[]`. The array would contain 63 `Long` numbers and the rest would be `EnsoBigInteger` instances that also extend `java.lang.Number`.